jwst/extract_1d/tests/test_apply_apcorr_nonifu.py

```diff
@@ -5,12 +5,12 @@
 from astropy.io import fits
 from astropy.table import Table
 
-from jwst.datamodels import JwstDataModel
-from jwst.extract_1d.apply_apcorr import ApCorr, ApCorrRadial, ApCorrPhase, select_apcorr
+from jwst.datamodels import DataModel
```

Comment:
Hmmm, this might need to be `JwstDataModel`, because I think `DataModel` went away with the migration to using the new `stdatamodels` package. Need to confirm.